spk/sickrage/src/installer.sh

```diff
@@ -59,9 +66,13 @@ postinst ()
     ${VIRTUALENV} --system-site-packages ${INSTALL_DIR}/env > /dev/null
 	
     # Clone the repository, install requirements and configure autoProcessTV
-    ${GIT} clone -q https://git.sickrage.ca/sickrage/sickrage.git ${INSTALL_DIR}/var/SickRage
+    ${GIT} clone --depth 10 --recursive -q -b ${wizard_fork_branch:=master} ${wizard_fork_url:=https://git.sickrage.ca/sickrage/sickrage.git} ${INSTALL_DIR}/var/SickRage > /dev/null 2>&1
```

Comment:
uhm... seems you are installing SickRage in the var folder... that means it will be backed up and overwrite older installations when updating the SPK since the var folder is backuped and restored after install is done.

Most packages install their app in the share folder instead. var is mostly used as data dir (saving DB, configs etc)

Reply:
So far every example I've seen uses the var folder when installing via GIT, I'm happy to learn something new if you want to show me but this is how CouchPotato and SIckBeard have been doing it as well

Reply:
yeah you are right. it's a git install package. nevermind